### ISSUE TITLE
Baking error: file too long

### ISSUE THREAD

**Issue by zimmicz** [ludovicchabant/PieCrust2#11]
```
Error loading page: ../../../../../usr/local/lib/python3.4/dist-packages/PieCrust-2.0.0a6_1_gd5f1747-py3.4.egg/piecrust/resources/theme/pages/_index.html
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/PieCrust-2.0.0a6_1_gd5f1747-py3.4.egg/piecrust/page.py", line 243, in load_page
    return _do_load_page(app, path, path_mtime)
  File "/usr/local/lib/python3.4/dist-packages/PieCrust-2.0.0a6_1_gd5f1747-py3.4.egg/piecrust/page.py", line 284, in _do_load_page
    cache.write(cache_path, json.dumps(cache_data))
  File "/usr/local/lib/python3.4/dist-packages/PieCrust-2.0.0a6_1_gd5f1747-py3.4.egg/piecrust/cache.py", line 85, in write
    with codecs.open(cache_path, 'w', 'utf-8') as fp:
  File "/usr/lib/python3.4/codecs.py", line 896, in open
    file = builtins.open(filename, mode, buffering)
OSError: [Errno 36] File name too long: '/home/michal/dev/www/posts.zimmi.cz/_cache/pages/__index__.._.._.._.._.._usr_local_lib_python3.4_dist-packages_PieCrust-2.0.0a6_1_gd5f1747-py3.4.egg_piecrust_resources_theme_pages__index.html.json'
Baking was aborted due to 1 error(s):
('Error loading page: /usr/local/lib/python3.4/dist-packages/PieCrust-2.0.0a6_1_gd5f1747-py3.4.egg/piecrust/resources/theme/pages/_index.html', OSError(36, 'File name too long'))
Baking was aborted due to errors.
```

using the latest git repo code, haven't seen this error before.

**Comment by ludovicchabant** [comment]
Weird, I thought I replied to this.
Looks like I need to change the way PieCrust generates temp file names... I'll look into it after I'm done battling various bugs in the incremental baking process. Thanks for reporting this!

**Comment by ludovicchabant** [comment]
Fixed with 566f5804f20d43e8cb0fdc68e0e485623384d455. Thanks!
